etna/transforms/math/statistics.py

```diff
@@ -100,7 +109,9 @@ def _transform(self, df: pd.DataFrame) -> pd.DataFrame:
 
     def get_regressors_info(self) -> List[str]:
         """Return the list with regressors created by the transform."""
-        return [self.out_column_name]
+        if self.in_column_regressor is None:
+            warnings.warn("Regressors info might be incorrect. Fit the transform to get the correct regressors info.")
```

Comment:
Why do we want to use warning? Is it ok to be in this state at all?

Reply:
It is not ok, but as it is a public method it might be called when the transform is not fitted. We do the same thing in the other transforms too, i.e [here](https://github.com/tinkoff-ai/etna/blob/17ca030bac06b132310bfe0fcb41b84e92593fdf/etna/transforms/math/add_constant.py#L139)

Reply:
In some transforms this method work correctly even in not fitted state, but here for the correct work it should be fitted.

Reply:
Don't you think it is more correct to raise an exception in this case?

Reply:
Fixed